### PR TITLE
Update README in preparation for version 1 release

### PR DESCRIPTION
**Pull request by gwaybio**
# Description

Reorganizing, rewording, and adding to README in preparation for version 1 release.

## What is the nature of your change?

- [ ] Bug fix (fixes an issue).
- [x] Enhancement (adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] This change requires a documentation update.

# Checklist

Please ensure that all boxes are checked before indicating that a pull request is ready for review.

- [x] I have read the [CONTRIBUTING.md](CONTRIBUTING.md) guidelines.
- [x] My code follows the style guidelines of this project.
- [x]  I have performed a self-review of my own code.
- [x]  I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x]  My changes generate no new warnings.
- [x] New and existing unit tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] I have deleted all non-relevant text in this pull request template.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,13 +133,14 @@ If you want to perform this data collation inside pycytominer using the `cyto_ut
...
 pip install "pycytominer[collate] @ git+git://github.com/cytomining/pycytominer@2aa8638d7e505ab510f1d5282098dd59bb2cb470"
 ```
 
 If using `pycytominer` in a conda environment, in order to run `collate.py`, you will also want to make sure to add `cytominer-database=0.3.4` to your list of dependencies.
 
-## Creating a cell locations lookup table
+### Creating a cell locations lookup table
 
 The `CellLocation` class offers a convenient way to augment a [LoadData](https://cellprofiler-manual.s3.amazonaws.com/CPmanual/LoadData.html) file with X,Y locations of cells in each image.
 The locations information is obtained from a single cell SQLite file.
```
Comment (kenibrewer) [inline README.md:146]: Install instructions in line 152 should also be changed to match above approach.

**Review by kenibrewer — COMMENTED** [review]
I'd recommend a few changes in the alternative installation instructions:

File: README.md [hunk 2]
```diff
@@ -12,41 +12,117 @@ The tool is most often used for processing data through the following pipeline:
...
+conda install pycytominer
+```
 
-Pycytominer is still in beta, and can only be installed from GitHub:
+Since the project is actively being developed, you can also install using the most up to date (or an older) github hash
 
 ```bash
-pip install git+git://github.com/cytomining/pycytominer
+# With ssh keys enabled
+pip install git+git://github.com/cytomining/pycytominer@77d93a3a551a438799a97ba57d49b19de0a293ab
```
[the remark this answers is not in the document]
Comment (gwaybio) [reply README.md:40]: Thanks for this discussion @kenibrewer 

My initial thought with this was to add these instructions for a user scenario where they need new pycytominer functionality. After thinking this through, we should not support this kind of scenario moving forward, after version 1.0, where we should develop a new, more sustainable, release process.

So I will go with your option 1 `Delete the pip install git+git: example entirely` as it is the simplest and supports our long term strategy.

File: README.md [hunk 3]
```diff
@@ -57,13 +133,14 @@ If you want to perform this data collation inside pycytominer using the `cyto_ut
 ```bash
 # Example for general case commit:
 pip install "pycytominer[collate] @ git+git://github.com/cytomining/pycytominer"
```
[the remark this answers is not in the document]
Comment (gwaybio) [reply README.md:135]: yes I believe so. There is another example of this in cell_locations, so I will commit on my end

**Comment by gwaybio** [comment]
Thanks for the review @kenibrewer !

Are you able to approve the PR? It seems you went through thoroughly, so I am happy to merge after having your eyes on it.

**Review by kenibrewer — APPROVED** [review]

**Comment by kenibrewer** [comment]
@gwaybio I approved the changes, but I don't count as a reviewer with write access. I assume you can probably merge anyways though.

**Pull request merged**